tests/test_gcvit.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import tensorflow as tf
+from skimage.data import chelsea
+
+from gcvit import GCViTTiny
```

Comment:
To reduce CI time we can try smaller models like `GCViTXXTiny`. Output feature or feature map may change